bb-cumber/src/main/java/com/cognifide/qa/bb/cumber/guice/BobcumberModule.java

```diff
@@ -20,26 +20,37 @@
 package com.cognifide.qa.bb.cumber.guice;
 
 import com.cognifide.qa.bb.assertions.soft.SoftAssertionsAspect;
+import com.cognifide.qa.bb.loadable.CucumberLoadableProcessorFilter;
+import com.cognifide.qa.bb.loadable.LoadableProcessorFilter;
 import com.google.inject.AbstractModule;
 import com.google.inject.matcher.Matchers;
 
+import com.google.inject.multibindings.Multibinder;
 import cucumber.api.java.en.And;
 import cucumber.api.java.en.But;
 import cucumber.api.java.en.Given;
 import cucumber.api.java.en.Then;
 import cucumber.api.java.en.When;
 
+import static com.google.inject.matcher.Matchers.any;
```

Comment:
static imports should probably be at the top. see https://github.com/Cognifide/bobcat/blob/master/contributing/intellij-imports-order.png